### DISCUSSION TITLE
error on dial go install ./cmd/...

### DISCUSSION THREAD

**Discussion by Jimmy99** [benbjohnson/wtf#24] [Q&A] [answered]
Trying to run the dial app I get the following error on go install ./cmd/...

![error](https://user-images.githubusercontent.com/10570311/106612460-c96ce980-6571-11eb-9300-504e9b1688d5.png)
Does anyone know what the fix is for this error?

**Comment by moorereason** [comment, marked as answer]
It looks like you're using Go 1.15.6.  Package `io/fs` will be introduced in Go 1.16.  You can test the [pre-releases](https://golang.org/dl/#unstable) until Go 1.16 is finalized.

**Reply by Jimmy99** [reply]
Thanks for the response!
